Document AD annotations v2 (Agent 7.36+) (#13807)
DataDog/documentation: content/en/agent/kubernetes/integrations.md

metadata:
  name: '<POD_NAME>'
  annotations:
    ad.datadoghq.com/<CONTAINER_IDENTIFIER>.checks: |
      {
        "<INTEGRATION_NAME>": {
          "init_config": <INIT_CONFIG>,
          "instances": [<INSTANCE_CONFIG>]
        }
      }
    # (...)
spec:
  containers:
    - name: '<CONTAINER_IDENTIFIER>'
# (...)
```

> `init_config` is usually an empty `{}`, and with AD Annotations v2 it became optional.

Reviewer: ```suggestion
`init_config` is usually an empty `{}`. In AD Annotations v2, it is optional.
```

{{% /tab %}}
{{< /tabs >}}

### Tolerate unready pods

> By default, `unready` pods are ignored when the Datadog Agent schedules checks, so metrics, service checks, and logs are not collected from these pods. To override this behavior, set the annotation `ad.datadoghq.com/tolerate-unready` to `"true"`. For example:

Reviewer: ```suggestion
By default, `unready` pods are ignored when the Datadog Agent schedules checks. Therefore, metrics, service checks, and logs are not collected from these pods. To override this behavior, set the annotation `ad.datadoghq.com/tolerate-unready` to `"true"`. For example:
```
